src/lib/description-helpers/prompts.ts

```diff
@@ -0,0 +1,31 @@
+export const endpointSystemPrompt = `You are an expert, articulate programmer. In 1 sentence and a maximum of 20 words explain how the API endpoint is used in the context of the endpoint based on the details provided.
```

Comment:
`explain how the API endpoint is used in the context of the endpoint based on the details provided`
This text can be improved.

Imagine you have no context whatsoever about what someone is asking you to do, and they give you this as the instructions - do you know what they mean?

Reply:
I would also be careful to make it clear that the user message contains example parameters and return object which shouldn't be explicitly included in the description.

On my 2nd try, it gave the description of "called with parameter query=github" for searching google for "github". You don't want the AI to describe the endpoint as having been called with specific parameters